<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventBaseWidget</class>
 <widget class="QWidget" name="EventBaseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">EventBaseWidget{border:0px; 
                            background:white; 
                            font:Microsoft YaHei UI 12px;}
                        EventBaseWidget:hover{background:rgb(245,246,246);}</string>
  </property>
  <widget class="QLabel" name="eventIconWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>50</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="eventIcon">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>30</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="eventNameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>13</y>
     <width>550</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>EventName</string>
   </property>
  </widget>
  <widget class="QLabel" name="eventLocationLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>63</y>
     <width>650</width>
     <height>29</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>Event Type / Event Location | Event Date</string>
   </property>
  </widget>
  <widget class="QWidget" name="descriptionPoint" native="true">
   <property name="geometry">
    <rect>
     <x>715</x>
     <y>45</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
